examples/pulse_control/5_Build_single_qubit_gates.ipynb

```diff
@@ -0,0 +1,538 @@
+{
```

Comment:
<p>Again, this is a $48 bill. Lets put a disclaimer at the top telling customers how much this will cost if they run all the cells. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/294/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>